core/files/css/base.php

```diff
@@ -440,7 +440,7 @@ public function parse_property_placeholder( array $control, $value, array $contr
 			if ( ! empty( $control['responsive'] ) && ! empty( $controls_stack[ $parser_control_name ]['responsive'] ) ) {
 				$device_suffix = Controls_Manager::get_responsive_control_device_suffix( $control );
 
-				$control = $controls_stack[ $parser_control_name . $device_suffix ];
+				$control = $controls_stack[ $parser_control_name . $device_suffix ] ?? $controls_stack[ $parser_control_name ];
```

Comment:
When you want to set a condition that depends only on other's control desktop value, it won't work